src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -159,7 +121,24 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
-    private async _processAdditionalAction(error: StripeAdditionalActionError, methodId: string, shouldSaveInstrument: boolean): Promise<any> {
+    private _buildPaymentPayload(methodId: string, formattedPayload: any, shouldSetAsDefaultInstrument: boolean): any {
```

Comment:
It's better to avoid the use `any` instead you can use `unknown` as it is less permissive